frontend/src/containers/Header/Header.module.scss

```diff
@@ -2,8 +2,8 @@
 @import '/src/styles/mixins';
 
 .headerWrapper {
-  padding-top: 30px;
-  @include flex(row, center, center, nowrap);
+  padding: 16px;
+  @include flex(row, space-between, center, nowrap);
```

Comment:
Try to use this mixin for spacing like: padding: spacing(2);




$defaultSpacingUnit: 8;
@function spacing($top, $right: null, $bottom: null, $left: null) {
  @if $top != null and $right == null and $bottom == null and $left == null {
    @return #{$top * $defaultSpacingUnit}#{'px'};
  }

  @if $top != null and $right != null and $bottom == null and $left == null {
    @return #{$top * $defaultSpacingUnit}#{'px'} #{$right * $defaultSpacingUnit}#{'px'};
  }

  @if $top != null and $right != null and $bottom != null and $left == null {
    @return #{$top * $defaultSpacingUnit}#{'px'} #{$right * $defaultSpacingUnit}#{'px'} #{$bottom * $defaultSpacingUnit}#{'px'};
  }

  @return #{$top * $defaultSpacingUnit}#{'px'} #{$right * $defaultSpacingUnit}#{'px'} #{$bottom * $defaultSpacingUnit}#{'px'}
    #{$left * $defaultSpacingUnit}#{'px'};
}